CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2021-06-28
```

Comment:
Since we're don't know the actual release date, typically people would but In-Development until we're about to merge into main.

Reply:
Good to know. 

I was going to leave out the ChangeLog version bumping until the PR from the sprint branch into dev was done, but I wanted to ensure I was at least tracking changes along the way.